app/representers/api/min/account_representer.rb

```diff
@@ -39,4 +39,12 @@ class Api::Min::AccountRepresenter < Api::BaseRepresenter
     }
   end
   embed :stories, paged: true, item_class: Story, item_decorator: Api::Min::StoryRepresenter, zoom: false
+
+  link :series do
+    {
+      href: "#{api_account_series_index_path(represented)}{?filters}",
```

Comment:
Not sure why I made the `min` version of this link not match the regular representer, but I think they probably should both include paging (as should the link to pieces in the min).